app/src/main/java/tech/pauly/findapet/data/PetfinderEndpoints.java

```diff
@@ -0,0 +1,17 @@
+package tech.pauly.findapet.data;
+
+import tech.pauly.findapet.BuildConfig;
+
+public class PetfinderEndpoints {
+    public static String getEndpoint() {
+        String environment = BuildConfig.ENVIRONMENT;
```

Comment:
In the future probably want to wrap this in an injectable object for testing, but I'm not one to test a simple switch statement, so it's fine for now.